src/components/Footer.js

```diff
@@ -1,8 +1,127 @@
-import React from "react";
+import React from 'react';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faLinkedin, faGithub } from '@fortawesome/free-brands-svg-icons';
 
 const Footer = () => {
+  const currentYear = () => {
+    const today = new Date();
+    const year = today.getFullYear();
+    return year;
+  };
+
   return (
-    <h1> Hello This is Footer component </h1>
+    <footer className="bg-primary text-white pt-3">
+      <div className="d-flex justify-content-around">
+        <section>
+          <h6 className="text-uppercase">Social</h6>
+          <ul className="list-unstyled mb-1">
+            <li>
+              <p className="mb-0">
+                <a
+                  className="github social text-white"
```

Comment:
The `github` class name is no longer used so it should be removed. Also, `social` can be renamed to the new selector `footer-link`

Reply:
I also removed the `linkedin` class name I think it also need to be removed